docs/actions/introduction.md

```diff
@@ -0,0 +1,171 @@
+# Actions
+
+[^1]: [Here, you can see the code for all actions.](https://github.com/chutney-testing/chutney/tree/master/task-impl/src/main/java/com/chutneytesting/task)
+
+
+Chutney provides a set of capabilities, or `Actions`, you can use in your scenarios.
+They replace all the boilerplate code you would have to write and maintain for executing your scenarios.
+You can see them as a set of small generic clients. [^1]
+
+For example, instead of writing your own HTTP client for doing a POST request, you just have to use the [HttpPost](/actions/http/#post) action
+and give it the minimum amount of information as inputs (i.e. targeted service, URI, body and headers).
+
+All actions are structured the same way with **inputs**, **outputs**, **validations** and **teardown**.
+
+## Inputs
+
+Inputs are the minimum information needed to run the action.  
+For example, if you want to perform an HTTP GET request, you should give, at least, the targeted service and an URI.  
+Obviously, you should be familiar with the technology behind each action you use, and we stick to the proper vocabulary (i.e. _body_ for HTTP, _payload_ for Kafka etc.)
+
+!!! note
+    Some input values are required and checked for correctness. While other values might not be required, or we provide a default value.
+
+!!! note
+    All actions performing a request on a remote service need to know the `Target`. While other action, like validating XML data, don't need a target.
+
+## Outputs
+
+Outputs contain the data collected after performing an action.
```

Comment:
you could add, output are evaluated only of step is OK

Reply:
done